lib/maplibre.ex

```diff
@@ -173,6 +180,13 @@ defmodule MapLibre do
     end
   end
 
+  defp validate_geojson!(opts) do
+    if !Keyword.get(opts, :data) do
```

Comment:
Same here:

```suggestion
    if !opts[:data] do
```

Reply:
Also you could use `unless` or `is_nil`:

```suggestion
    if is_nil(opts[:data]) do
```